.github/workflows/changelog_verifier.yml

```diff
@@ -6,7 +6,10 @@ on:
 jobs:
   # Enforces the update of a changelog file on every pull request
   verify-changelog:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ 'ubuntu-22.04', 'ubuntu-20.04', 'ubuntu-latest' ]
```

Comment:
Thanks for the PR @Yury-Fridlyand! Just wanted to understand is there any specific reason for picking up `22.04` and `20.04` version of ubuntu?